ci-operator/config/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main__aws-4.14-nightly-x86.yaml

```diff
@@ -93,12 +93,14 @@ tests:
 - as: perfscale-aws-data-path-9nodes-periodic
   cron: 0 21 * * 2
   steps:
-    cluster_profile: aws-perfscale-qe
+    cluster_profile: aws-perfscale
```

Comment:
@jtaleric Any reason why this is perfscale when # of nodes are less than 24?

Reply:
@svetsa-rh When I tried this using the QE account, I had errors launching this specific instance type `c5.4xlarge` not sure if we need special quota for the `c5` but since this is a small test that completes in a reasonably fast time, i didn't see the need to burden the QE Team to request quota.